### PR TITLE
Add left-turn command maze side

### PR DESCRIPTION
**Pull request by hannahbergam**
This is the corresponding PR to https://github.com/code-dot-org/code-dot-org/pull/40861 for the turnLeft() command. 

It required making a custom scheduleTurn() and a custom animatedTurn(), because the existing methods don't work with cardinal directions, and assume the avatar spritesheet has additional frames for turning the pegman.

https://user-images.githubusercontent.com/37230822/120400255-5e9aa780-c2f2-11eb-9647-a463b57fd5f4.mp4

Jira Ticket: https://codedotorg.atlassian.net/browse/CSA-357

### CODE REVIEW

File: src/animationsController.js [hunk 1]
```diff
@@ -392,6 +392,25 @@ module.exports = class AnimationsController {
     });
   }
 
+  /**
+   * Schedule the animations for a turn to the given direction.
```
Comment (molly-moen) [inline src/animationsController.js:396]: nit: explain here what a simple turn is

**Review by molly-moen — APPROVED** [review]
Looks great! One comment nit

**Review by jmkulwik — APPROVED** [review]
Nice work!

File: src/animationsController.js (hunk 1 above)
Comment (hannahbergam) [reply src/animationsController.js]: done! Thanks:)

**Pull request merged**